experiment/contracts/Crosschain.sol

```diff
@@ -0,0 +1,80 @@
+pragma solidity >=0.4.21 <0.6.0;
+pragma experimental ABIEncoderV2;
+
+import {Verifier} from "./Verifier.sol";
+
+contract Crosschain {
+    struct Event {
+        bytes20 receivingPKH;
+        uint256 amount;
+        bytes32 txID;
+    }
```

Comment:
We don't support multiple burn outputs per tx?

Reply:
we do